core/src/main/scala/chisel3/properties/Property.scala

```diff
@@ -323,6 +323,9 @@ sealed trait Property[T] extends Element { self =>
 
   final def +(that: Property[T])(implicit ev: PropertyArithmeticOps[Property[T]], sourceInfo: SourceInfo): Property[T] =
```

Comment:
```suggestion
  /** Perform addition as defined by FIRRTL spec section 25.1.1. (as of 4.0.0) */
  final def +(that: Property[T])(implicit ev: PropertyArithmeticOps[Property[T]], sourceInfo: SourceInfo): Property[T] =
```

or maybe there is a more future-proof way of referring to that